data/mods/package_bionic_professions/bionic_professions.json

```diff
@@ -698,7 +705,7 @@
       "bio_sleepy",
       "bio_trip",
       "bio_spasm",
-      "bio_metabolics",
+      "bio_lampoil",
```

Comment:
I'm not sure that lampoil makes sense for this character? Maybe the trickle charge?

Reply:
I was looking for some character to put lampoil on, and it came down to either cyberjunkie or bionic prepper. I decided on the cyberjunkie because they were meant to have eclectic, cobbled-together CBMs so the lamp oil being "weird" felt fitting to me. I can leave them as metabolic though.

Reply:
That makes sense I guess.